provider-ui/components/ProviderLayout.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import NoSsr from '@mui/material/NoSsr'
+
+export default function ProviderLayout({ children }) {
+  return (
+    <>
+      <NoSsr>
+        <div data-cy="root" style={{ padding: '170px 0px', textAlign: 'center' }}>
```

Comment:
If we're rewriting, then let's not propagate bad practices of static styling, but instead move to flex.